app/assets/javascripts/index.js

```diff
@@ -0,0 +1,235 @@
+// ESLINT:
+/* global L, moment, $ */
+
+
+// Load all the channels within this directory and all subdirectories.
+// Channel files must be named *_channel.js.
+
+// Const channels = require.context('.', true, /_channel\.js$/)
+// Channels.keys().forEach(channels)

```

Comment:
This commented code and the comment above it seem like dead code? Can they be deleted?